payjoin/tests/integration.rs

```diff
@@ -33,7 +33,11 @@ mod integration {
 
             // Receiver creates the payjoin URI
             let pj_receiver_address = receiver.get_new_address(None, None)?.assume_checked();
-            let pj_uri = build_pj_uri(pj_receiver_address, Amount::ONE_BTC, EXAMPLE_URL, None);
+            let pj_uri =
+                PjUriBuilder::new(pj_receiver_address, payjoin::Url::parse(EXAMPLE_URL).unwrap())
```

Comment:
Similar to the comment asking for the `pj_endpoint` to be parsed earlier, perhaps we make the EXAMPLE_URL a `static` variable that's already a Url. I'd also rather it's imported than qualified especially since it's in-line.